### DISCUSSION TITLE
In today's 4.0.1 release, the displayed SHA256 checksum for `SubtitleEdit-4.0.1-Setup.exe` is that of 4.0.0

### DISCUSSION THREAD

**Discussion by darkred** [SubtitleEdit/subtitleedit#7399] [General]
As the title says, in today's release, [4.0.1](https://github.com/SubtitleEdit/subtitleedit/releases/tag/4.0.1), the actual SHA256 checksum of `SubtitleEdit-4.0.1-Setup.exe` is:
`124a9dd46e041ed5fc05dc6341cf9ecc25ed5c7fa0b2512e42c20ff53240f005`.

The displayed one, `0b7640d9c2a58e163aa69e6e6c0ab94ee735a2281628b6ed3e42eeb4a3a45862`, is of `SubtitleEdit-4.0.0-Setup.exe`.

Kindly cc @niksedk

**Comment by niksedk** [comment]
Thx :)
Fixed

**Comment by Blackbird52** [comment]
SHA256 Checksum for SE401.zip is wrong, I download it 2 time and my hash is 3729606B10A6CC47BC72C6FF1570931EAF1072974F979137EA6239ED6EDADA48
please verify

**Comment by niksedk** [comment]
Fixed - I must double check the hashes in next release...
